File: java/app/src/main/java/tw/com/ecpay/paymentgatewaykit/example/api/ConsumerInfo.java
package tw.com.ecpay.paymentgatewaykit.example.api;

public class ConsumerInfo {
    public String MerchantMemberID;
    public String Email;
    public String Phone;
    public String Name;
    public String CountryCode;
    public String Address;
    public ConsumerInfo(String MerchantMemberID,
                        String Email,
                        String Phone,
                        String Name,
                        String CountryCode,
                        String Address) {
        this.MerchantMemberID = MerchantMemberID;
        this.Email = Email;
        this.Phone = Phone;
        this.Name = Name;
        this.CountryCode = CountryCode;
        this.Address = Address;
    }
}

File: kotlin/app/src/main/java/tw/com/ecpay/paymentgatewaykit/example/gateway/api/ConsumerInfo.kt
package tw.com.ecpay.paymentgatewaykit.example.gateway.api

data class ConsumerInfo(
    var MerchantMemberID: String,
    var Email: String,
    var Phone: String,
    var Name: String,
    var CountryCode: String,
    var Address: String
)